identifiers/country-pl/provinces.csv

```diff
@@ -0,0 +1,18 @@
+id,name
+ocd-division/country:pl,Polska
+ocd-division/country:pl/province:ds,Dolnośląskie
```

Comment:
Similar to Portugal, please use "/state:" as the canonical type for the first level of administrative areas inside a country, and "/province:" in the aliases.csv file.

Reply:
alright

Reply:
should be fixed